src/content/stateDashContent.ts

```diff
@@ -3267,3 +3267,26 @@ export const stateData = [
   'Wisconsin',
   'Wyoming',
 ].sort();
+
```

Comment:
create a type for data links:
```
type DataLinks = {
  excessDeathRates: string[];
  covid19DeathRates: string[];
  potentialUndercounting: string[];
  medicalDeathInves: string[];
};
```
then replace the below line 3271 with:
```
export const dataLinks: DataLinks = {
```

With this `DataLinks` type, you don't need the `DataType` inside the `countyDashboard.tsx` and replace those with `DataLinks`. This is the same for countyData above. You could create a type for that just like how did for datalink.